src/coreclr/debug/runtimeinfo/runtimeinfo.cpp

```diff
@@ -13,7 +13,7 @@
 
 // Runtime information public export
 #ifdef HOST_UNIX
-DLLEXPORT
```

Comment:
I am not sure I understand this change. Unless I am missing something, the DLLEXPORT is defined as `__attribute__((visibility("default")))`.

Reply:
Hmm. I think it did not work or was not defined in the context of this file. I will try reverting this line change.

Reply:
Works now with `DLLEXPORT`. I am not sure where it is defined, since this file seems to include only `windows.h`, but somehow it works now.

Reply:
> this file seems to include only `windows.h`

This `windows.h` is actually the one in runtime sources, which includes this chain of headers: `palrt.h->pal.h->palinternal.h->mbusafecrt_internal.h` to obtain the definition from:
https://github.com/dotnet/runtime/blob/69e114c1abf91241a0eeecf1ecceab4711b8aa62/src/coreclr/pal/src/safecrt/mbusafecrt_internal.h#L24